### DISCUSSION TITLE
blog/2024-in-review

### DISCUSSION THREAD

**Discussion by giscus** [TkDodo/blog-comments#115] [Announcements]
# blog/2024-in-review

A retrospective on what has happened for me in 2024

https://tkdodo.eu/blog/2024-in-review

**Comment by tigerabrodi** [comment]
Hell of a year, the suspense gate thing was funny to see

but also cool to see how receptive the react team was

i think youll do fantastic at Sentry, grats

**Reply by TkDodo** [reply]
Thank you 🙏. Happy new year to you too, and hopefully we'll meet in person next year 🚀

**Comment by riskers** [comment]
Wishing you a better and better New Year
